languages/go/reference/README.md

```diff
@@ -27,12 +40,19 @@
 - [Boolean logic](../../../reference/concepts/boolean_logic.md)
 - Channels
 - Code Formatting
+- context.Context
 - [Comments](../../../reference/concepts/comments.md)
 - Concurrency
 - [Conditionals](../../../reference/concepts/conditionals.md)
+- CSV Handling
```

Comment:
I feel that this is at the edge of what can be still seen as a Concept. Is this a core Go (language) feature? I'm leaning towards this being a bit too specific to be taught in Concept Exercises.

Reply:
Agree. Will take it out.